pysat/instruments/dmsp_ivm.py

```diff
@@ -243,6 +258,13 @@ def smooth_ram_drifts(inst, rpa_flag_key=None, rpa_vel_key='ion_v_sat_for'):
 
     """
 
+    warnings.warn("".join(["This funciton is deprecated here and will be ",
```

Comment:
```suggestion
    warnings.warn("".join(["This function is deprecated here and will be ",
```
and elsewhere in this file.